### PR TITLE
Make subset of python files PEP8-compatible

### PR DESCRIPTION
**Pull request by vitalii-komisarenko**
Making non-test-related python file pep8-compatible.

### CODE REVIEW

File: tools/commands/deploy.py
Comment (arnout) [inline tools/commands/deploy.py:47]: Maybe we should decide on a coding style for this kind of situation...

```
        "blahblahblah".format(
            x, y, z)
```

or

```
        "blahblahblah".\
            format(x, y, z)
```

Both are a bit ugly, but I have a slight preference for the second option.

Well, actually, I'd really prefer to use f-strings which would remove the need for ever using `"...".format()`, but apparently not everyone has python 3.6 :-(

File: tools/commands/deploy.py
Comment (arnout) [inline tools/commands/deploy.py:139]: Meh, it's a bit silly to fix this since it actually doesn't work any more... There are no longer any modules...

Oh well, let's just fix it for pep8.

File: tools/commands/config.py
Comment (arnout) [inline tools/commands/config.py:126]: The previous commit still leaves a catch-all except, so why not do the same here?

File: tools/commands/build.py
Comment (arnout) [inline tools/commands/build.py]: The double newline should be *before* the command, the comment is tied to the cmakebuilder class.

File: tools/commands/build.py
Comment (arnout) [inline tools/commands/build.py]: Same here, the comment is tied to the acbuilder class.

File: tools/commands/build.py
Comment (arnout) [inline tools/commands/build.py]: And same here again.

File: tools/commands/build.py
Comment (arnout) [inline tools/commands/build.py]: I don't think this is a nice way to split. Better split on the else:

```suggestion
        _dep_modules = dep_modules if 'all' in args.modules or 'dep' in args.modules 
            else [m for m in dep_modules if m in args.modules]
```

**Review by arnout — APPROVED** [review]
I think it would be good to merge this soon. There's no need to have everything PEP8 compliant before we merge this.

Also, it may be a good idea to already add the pep8 check to CI, but just allow it to fail.

**Comment by adam1985d** [comment]
> Making deploy.py PEP8-compartible as a part of
> preparation for automatic style checks of python files on CI.
> 
> Signed-off-by: Vitalii Komisarenko [vitalii.komisarenko@gmail.com](mailto:vitalii.komisarenko@gmail.com)

I think you meant `compatible` and not `compartible` - both in PR description and in PR name

this comment is also relevant to all commit headlines and messages

File: tools/commands/config.py
Comment (vitalii-komisarenko) [reply tools/commands/config.py:126]: flake8 compains with `E722 do not use bare except`, so I add `Exception`

**Comment by vitalii-komisarenko** [comment]
beerocks_message_endian_converter.py is a python2 file.
Shall it be converted to python3?

**Comment by tomereli** [comment]
> beerocks_message_endian_converter.py is a python2 file.
> Shall it be converted to python3?

I think we can delete it. Right @vitalybu ?

File: tools/beerocks_analyzer/beerocks_analyzer.py
Comment (rmelotte) [inline tools/beerocks_analyzer/beerocks_analyzer.py]: What a mess... This shouldn't even be here, it should be handled by a library like argparse.

For now if you have the opportunity, could you just remove all the extra spaces before the second dash? So the first one would be like this for example:
```
"    -f=<input log file> - input log file"
```
It should make it look a tiny bit less messy.

File: tools/beerocks_analyzer/beerocks_analyzer_widget.py
Comment (rmelotte) [inline tools/beerocks_analyzer/beerocks_analyzer_widget.py]: Same here, if you have the time/opportunity, please remove all the extra spaces after the single dash.

**Review by rmelotte — APPROVED** [review]
Pretty hard to review in details of course, but overall LGTM.

Can you also add the ones that passes flake8 to `FLAKE8_FILES` in `.gitlab-ci.yml`?

**Comment by vitalybu** [comment]
> > beerocks_message_endian_converter.py is a python2 file.
> > Shall it be converted to python3?
> 
> I think we can delete it. Right @vitalybu ?

Yup

File: tools/commands/deploy.py
Comment (vitalii-komisarenko) [reply tools/commands/deploy.py:139]: Fixed for PEP8

File: .gitlab-ci.yml
Comment (arnout) [inline .gitlab-ci.yml:31]:
```suggestion
    FLAKE8_FILES: "tests/*.py tools/beerocks_analyzer/conf/conf1.py tools/commands/*.py"
```

Comment (vitalii-komisarenko) [reply .gitlab-ci.yml:31]: beerocks_analyzer already passes the code check.
typo commEnds->commAnds is addressed in https://github.com/prplfoundation/prplMesh/pull/1118

**Pull request merged**
